Update to Alpine Linux 3.19 and improved README to show size differences for alpine versions.

parasew opened the pull request.

parasew committed:
Update to alpine3.17 for Dockerfile.

parasew committed:
Update README.md

Adding size differences of the different Alpine versions.

parasew committed:
Update README.md

Creating a table for readability.

parasew committed:
alpine3.18

parasew committed:
added more stats and alpine 3.19

parasew committed:
Alpine 3.19

kpcyrd commented on Dockerfile:
```diff
@@ -1,5 +1,5 @@
 # Start with a rust alpine image
-FROM rust:1-alpine3.16
+FROM rust:1-alpine3.15
```
```suggestion
FROM rust:1-alpine3.19
```

kpcyrd commented:
Thanks! I'm not sure if historic data is relevant though, using end-of-life Alpine/Rust releases is not adviced (even if they might be slightly smaller).

parasew committed:
Alpine 3.19

parasew committed:
advising against using eol alpine versions

parasew commented:
I changed the Alpine version to 3.19, as you suggested. I also added your comment about not using older (eol) Alpine versions, even if they are smaller in size. The historic table is relevant imho, as it shows how Alpine docker image sizes have changed over the course of the last versions. I see the whole repo as an educational resource (so it can help people to learn about Docker, Alpine and Rust). Hope this suits.

kpcyrd commented:
Thanks!
